tests/integration.test.ts

```diff
@@ -0,0 +1,17 @@
+import { RawRequest } from "./utils/requestUtils";
+import { getStatusCode } from "./utils/errors";
+
+import { callRequests } from "./callRequests";
+
+// test("execute simple-get GOT request", async () => {
+//   const response = await executeGotRequest(got("https://postman-echo.com/get", { method: "GET" }));
+//   expect(response.byteLength).toBeGreaterThan(0);
+//   expect(response.executionTime).toBeGreaterThan(0);
+//   expect(response.error.length).toBeLessThan(1);
+// });
+
+test("execute auto-tests.zzb in default env", async () => {
+  const rawReq = new RawRequest("./tests/bundles/auto-tests.zzb", "default");
+  await callRequests(rawReq);
+  expect(getStatusCode()).toBe(0);
```

Comment:
Can we not just return the number of tests passed from callRequests()? Keeping it in a global variable, that too under `process.exitCode` seems inappropriate.

Reply:
I'm currently incrementing the status code for any errors encountered during the of a particular request, so the status code would effectively reflect the number of failed requests. This is copied over from the CLI, though. I can make this return the number of tests passed instead if required, and assert that that is 0.

Reply:
To be more clear: any failure associated with a request (whether it is an unintended pass or fail), is appended to an array, which is then displayed as a list after the name of the request. If the length of this array is more than 0, then I increment `process.exitCode` to denote that a failure has occurred.